build_projects/shared-build-targets-utils/Utils/Crossgen.cs

```diff
@@ -154,15 +154,9 @@ public void CrossgenDirectory(string sharedFxPath, string pathToAssemblies)
 
                 IList<string> crossgenArgs = new List<string> {
                     "-readytorun", "-in", file, "-out", tempPathName,
+                    "-JITPath", GetLibCLRJitPathForVersion(),
                     "-platform_assemblies_paths", platformAssembliesPaths
                 };
-
```

Comment:
Your PR is failing because you are removing this "good" change in core-setup.  The existing code needs to be ported into CLI, which is being done here:  https://github.com/dotnet/cli/pull/3223